fuzz/build.info

```diff
@@ -13,6 +13,7 @@ IF[{- !$disabled{"fuzz-afl"} || !$disabled{"fuzz-libfuzzer"} -}]
   PROGRAMS{noinst}=punycode pem decoder hashtable acert
   PROGRAMS{noinst}=v3name
   PROGRAMS{noinst}=provider
+  PROGRAMS{noinst}=ml-kem
```

Comment:
I believe this needs to be guarded like this.

```
  IF[{- !$disabled{"ml-kem"} -}]
    PROGRAMS{noinst}=ml-kem
  ENDIF
```

Same thing below under the `IF[{- !$disabled{tests} -}]` on line 151

Should be similar to how I guarded quic-server fuzzer: https://github.com/openssl/openssl/pull/26514